src/plugins/GUITestBase/src/test_runner/GUITestRunner.cpp

```diff
@@ -107,6 +111,31 @@ void GUITestRunner::sl_runAllGUITests() {
     }
 }
 
+void GUITestRunner::sl_listToStdout()
```

Comment:
Move unrelated improvements into a separate patch so we can discuss them separately and merge faster